addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js

```diff
@@ -23,15 +23,11 @@ export class ProjectTaskKanbanRenderer extends KanbanRenderer {
 
     canCreateGroup() {
         // This restrict the creation of project stages to the kanban view of a given project
-        return super.canCreateGroup() && (this.isProjectTasksContext() == this.props.list.isGroupedByStage
-            && this.isProjectManager || this.props.list.groupByField.name === 'personal_stage_type_id');
-    }
-
-    isProjectTasksContext() {
         return (
-            ["project.project", "project.task.type.delete.wizard"].includes(
-                this.props.list.context.active_model
-            ) && !!this.props.list.context.default_project_id
+            super.canCreateGroup() &&
+            ((!!this.props.list.context.default_project_id == this.props.list.isGroupedByStage &&
+                this.isProjectManager) ||
+                this.props.list.groupByField.name === "personal_stage_type_id")
```

Comment:
```suggestion
            super.canCreateGroup() && ((
                !!this.props.list.context.default_project_id == this.props.list.isGroupedByStage && this.isProjectManager
            ) || this.props.list.groupByField.name === "personal_stage_type_id")
```
Looks more readable to me, but it doesn't really matter :shrug: